### PR TITLE
change the button title ons electing an option

### PR DESCRIPTION
**Pull request by 30mar**
## Description
Button text should dynamically change when changing the selected option,

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-4597

## Screenshots

https://user-images.githubusercontent.com/15270737/104566836-66f98b00-564e-11eb-8486-48ecc9afd052.MP4

https://user-images.githubusercontent.com/15270737/104566849-6a8d1200-564e-11eb-906c-46517e7446b9.MP4

### CODE REVIEW

**Review by SwissMountainFrog — APPROVED** [review]

**Review by marcussc — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionSymptomsViewController.swift [hunk 1]
```diff
@@ -91,10 +91,13 @@ final class ExposureSubmissionSymptomsViewController: DynamicTableViewController
 			switch index {
 			case 0:
 				selectedSymptomsOption = .yes
+				navigationFooterItem?.primaryButtonTitle = AppStrings.ExposureSubmissionSymptoms.continueButton
 			case 1:
 				selectedSymptomsOption = .no
+				navigationFooterItem?.primaryButtonTitle = AppStrings.ExposureSubmissionSymptomsOnset.continueButton
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionSymptomsViewController.swift:97]: Would be nice to have this as a separate AppString for this Screen, e.g. `AppStrings.ExposureSubmissionSymptoms.doneButton`

**Review by nickguendling — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionSymptomsViewController.swift (hunk 1 above)
Comment (30mar) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionSymptomsViewController.swift:97]: should i change the name of continue button or add a new variable for this? I'm not sure why it was called continue

Comment (nickguendling) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionSymptomsViewController.swift:97]: I would propose to add a new one called `AppStrings.ExposureSubmissionSymptoms.doneButton` that contains the "Fertig"/"That's it!" string, and keep the old one `AppStrings.ExposureSubmissionSymptoms.continueButton` that contains the "Weiter"/"Next" string.

Comment (30mar) [reply src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionSymptomsViewController.swift]: I changed it actually, for me doesn't make sense to have a "continue" button that says "done", or what do you think?

File: src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift [hunk 2]
```diff
@@ -241,7 +241,7 @@ enum AppStrings {
 		static let answerOptionOneToTwoWeeksAgo = NSLocalizedString("ExposureSubmissionSymptomsOnset_AnswerOptionOneToTwoWeeksAgo", comment: "")
 		static let answerOptionMoreThanTwoWeeksAgo = NSLocalizedString("ExposureSubmissionSymptomsOnset_AnswerOptionMoreThanTwoWeeksAgo", comment: "")
 		static let answerOptionPreferNotToSay = NSLocalizedString("ExposureSubmissionSymptomsOnset_AnswerOptionPreferNotToSay", comment: "")
-		static let continueButton = NSLocalizedString("ExposureSubmissionSymptomsOnset_ContinueButton", comment: "")
+		static let doneButton = NSLocalizedString("ExposureSubmissionSymptomsOnset_ContinueButton", comment: "")
```
Comment (tcfos) [inline src/xcode/ENA/ENA/Source/View Helpers/AppStrings.swift:244]:
```suggestion
		static let doneButton = NSLocalizedString("ExposureSubmissionSymptomsOnset_DoneButton", comment: "")
```

**Review by tcfos — CHANGES_REQUESTED** [review]
Please check AppStrings.ExposureSubmissionSymptoms.doneButton

**Review by nickguendling — APPROVED** [review]

**Review by tcfos — APPROVED** [review]

**Pull request merged**